src/EditorFeatures/CSharpTest/AddRequiredParentheses/AddRequiredParenthesesTests.cs

```diff
@@ -47,19 +81,6 @@ void M()
 }", RequireAllParenthesesForClarity);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddRequiredParentheses)]
-        public async Task TestNoArithmeticOnLowerPrecedence()
```

Comment:
Hmm, why was the test deleted?

Reply:
Exact duplicate of `TestArithmeticPrecedence`

Reply:
Ah got it, `TestArithmeticPrecedence` now also validates there is no diagnostic/code fix offered on `+`.